modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1404,6 +1407,93 @@ public String getName() {
         return visible;
     }
 
+    /**
+     * Indicates whether or not this {@code Node} is shown. A node is considered shown if it's
+     * part of a {@code Scene} that is part of a {@code Window} whose
+     * {@link Window#showingProperty showing property} is {@code true}. The {@link Node#visibleProperty visibility}
+     * of the node or its scene does not affect this property.
+     * <p>
+     * This property can be used in conjunction with {@link ObservableValue#when} to
+     * create bindings which are only actively listening to their source when the node is shown.
+     * <p>
+     * This property can also be used to start animations when the node is shown, and to stop them
+     * when it is no longer shown.
```

Comment:
This sentence sounds like `shownProperty()` is quite deeply connected to animations, which it isn't. Maybe you can clarify with "For example, ..."?

Reply:
I find it reads really weird when I just make it an example (without putting a code example below it); I only added these to indicate possible use cases.

Perhaps if I rephrase it to "This property can also be useful to perform actions when the node is shown or no longer shown, like starting and stopping animations".